platforms/jvm/plugins-jvm-test-suite/src/main/java/org/gradle/api/testing/toolchains/internal/JUnitJupiterToolchainParameters.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.testing.toolchains.internal;
+
+import org.gradle.api.provider.Property;
+
+/**
+ * Parameters for configuring a JUnit Platform test toolchain.
```

Comment:
```suggestion
 * Parameters for configuring a JUnit Jupiter test toolchain.
```

Should this javadoc be updated?

Reply:
Yes it should.  Copilot failed me.